game/missiongenerator/kneeboard.py

```diff
@@ -266,14 +268,33 @@ def _format_time(self, time: Optional[datetime.timedelta]) -> str:
         local_time = self.start_time + time
         return local_time.strftime(f"%H:%M:%S")
 
+    def _format_alt(self, alt: Distance) -> str:
+        if self.is_metric:
+            return f"{alt.meters:.0f} m"
+        else:
+            return f"{alt.feet:.0f} ft"
+
     def _waypoint_distance(self, waypoint: FlightWaypoint) -> str:
         if self.last_waypoint is None:
             return "-"
 
-        distance = meters(
-            self.last_waypoint.position.distance_to_point(waypoint.position)
-        )
-        return f"{distance.nautical_miles:.1f} NM"
+        if self.is_metric:
+            distance = meters(
+                self.last_waypoint.position.distance_to_point(waypoint.position)
+            )
+            return f"{(distance.meters / 1000):.1f} km"
+        else:
+            distance = meters(
+                self.last_waypoint.position.distance_to_point(waypoint.position)
+            )
+            return f"{distance.nautical_miles:.1f} nm"
+
+    def _waypoint_bearing(self, waypoint: FlightWaypoint) -> str:
+        if self.last_waypoint is None:
+            return "-"
+        bearing = self.last_waypoint.position.heading_between_point(waypoint.position)
```

Comment:
I was wondering if this was going to be accurate or not given map projections, and apparently it's not the first time I'd wondered that and this is in fact correct :) https://github.com/dcs-liberation/dcs_liberation/issues/915